src/views/v2/emails/purchase-receipt/greeting.php

```diff
@@ -0,0 +1,49 @@
+<?php
+/**
+ * Event Tickets Emails: Main template > Body > Greeting.
+ *
+ * Override this template in your own theme by creating a file at:
+ * [your-theme]/tribe/tickets/v2/emails/template-parts/body/greeting.php
+ *
+ * See more documentation about our views templating system.
+ *
+ * @link https://evnt.is/tickets-emails-tpl Help article for Tickets Emails template files.
+ *
+ * @version TBD
+ *
+ * @since TBD
+ *
+ * @var Tribe_Template   $this                  Current template object.
+ * @var Module           $provider              [Global] The tickets provider instance.
+ * @var string           $provider_id           [Global] The tickets provider class name.
+ * @var \WP_Post         $order                 [Global] The order object.
+ * @var int              $order_id              [Global] The order ID.
+ * @var bool             $is_tec_active         [Global] Whether `The Events Calendar` is active or not.
+ */
+
+if ( empty( $order ) ) {
+	return;
+}
+
+$hello = empty( $order['purchaser'] ) || empty( $order['purchaser']['first_name'] ) ? 
```

Comment:
Unfortunately we don't have access to the first name in many scenarios, so we'll need to rely on the full name here. We may need to update the designs for this.

Reply:
Well... can we build it into the handler where it splits the full name by space (`explode( ' ', $full_name )`), then uses the first element as the first name?

Reply:
Unfortunately, we don't know how people will submit their names. There's only one field for name. We've had the same problem when working on the app and figuring out how to sort the attendees by their last names (it was impossible). They might insert the last surname and then the first name in different languages. We also can't know if they have two first names, etc.